spec/jobs/emit_web_hook_event_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+require_dependency 'jobs/base'
+
+describe Jobs::EmitWebHookEvent do
+  before(:all) do
+    @post_hook = Fabricate(:web_hook)
+    @inactive_post_hook = Fabricate(:inactive_web_hook)
+    @category = Fabricate(:category)
+    @topic_hook = Fabricate(:web_hook)
+    @topic_hook.categories = [@category]
```

Comment:
You can do something like

`let(:topic_hook) { Fabricate(:web_hook, categories: [@category]) }`